### PR TITLE
#5:  Integrate the app with Tauri framework

### PR DESCRIPTION
**Pull request by alvinosh**
- #5

### CODE REVIEW

File: svelte.config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import adapter from '@sveltejs/adapter-vercel'
```
Comment (prokawsar) [inline svelte.config.js:1]: Hello @alvinosh 
PR looks nice.
May I know is that package `@sveltejs/adapter-vercel` made any trouble for tauri integration? 
That package used to deploy the app to web. 
https://ark-gallery.vercel.app/
It's not any important thing for this repo, just asking.

**Review by prokawsar — COMMENTED** [review]
@kirillt @alvinosh 
I think icons can be updated if we have any icons now.
Other than integration looks fine.

File: svelte.config.js (hunk 1 above)
Comment (alvinosh) [reply svelte.config.js:1]: For tauri to build the app it must be a static application, meaning the files should be generated ahead of time, thats why we cannot use the vercel adapter. If we need to deploy online we can deploy it like a regular static website

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          components: rustfmt, clippy
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3.2.0
+        with:
+          node-version: '16.x'
```
Comment (prokawsar) [inline .github/workflows/build.yml]: @alvinosh 
node-version: '20.x' would be fine for sveltekit I guess.

**Review by kirillt — APPROVED** [review]
Thanks, Alvi!

**Comment by kirillt** [comment]
@prokawsar we don't have icons yet, thanks for the idea.

**Comment by kirillt** [comment]
Follow-up issues:
- #7
- #8 
- #9

**Pull request merged**